### ISSUE TITLE
Runnable code environment pulling in next line after code

### ISSUE THREAD

**Issue by aamirzk** [chirun-ncl/chirun#160]
Using the runnable code environment in LaTeX pulls in the line of text after the code environment. 

<img width="869" alt="Screenshot 2023-07-27 at 12 02 00" src="https://github.com/chirun-ncl/chirun/assets/94388861/3e407eb3-e9f1-4151-8d30-9235dc4efb46">

**Comment by aamirzk** [comment]
Still having this issue with pulling in lines after \end{runnableCode}.

**Comment by christianp** [comment]
Can you give me a link to a package where this happens?

**Comment by aamirzk** [comment]
Towards the end of Chapter 3 of this [package](https://lti.chirun.org.uk/material/package/49125615-d284-4bb5-b904-ce0623bbbf8a/).

**Comment by christianp** [comment]
Ah, I don't think I updated the docker image which the production server uses. I'm doing that now, so I'll rebuild that package and see if it works.

**Comment by christianp** [comment]
Nope, that's not it. I'll have to have a closer look.

**Comment by christianp** [comment]
Works in my development environment, so this *must* be the Docker image being out of date. Arggg!!

**Comment by christianp** [comment]
Yes, I finally managed to rebuild the docker image properly: I'd given `--build-arg version=...` instead of `--build-arg VERSION=...`.
I've rebuilt your package and the runnable code block only contains what it should now.
